### DISCUSSION TITLE
autodiscover

### DISCUSSION THREAD

**Discussion by jfrancais** [EasyEngine/easyengine#1689] [Support Requests]
Looking to take advantage of autodiscovery for my websites.  The easiest way to implement set up https://MYDOMAIN.COM//autodiscover/autodiscover.xml to transparently proxy https://MYMAILSERVER.COM/autodiscover/autodiscover.xml.  I'm having trouble figuring where I can do this.  First it seems to redirect MYDOMAIN.COM to WWW.MYDOMAIN.COM.  Second, It seems to serve a 404 Wordpress page at that URL.

Any advise on where I could edit a config file to proxy an external URL in the above scenario?

**Comment by gingerbeardman** [comment]
WordPress works by routing all requests to your website.

So, you'll need to put an exception in so that it does not route your `autodiscover` folder and contents.

For EasyEngine this is done by modifying the nginx configuration.

1. `sudo su root`
2. `cd /var/lib/docker/volumes/global-nginx-proxy_confd/_data/`
3. edit the file `default.conf`

How to edit the file? Well, I'm not sure as I've reached the limit of my ability with EE/nginx.
